about.html

```diff
@@ -0,0 +1,120 @@
+                Attention music lovers!
+                We are thrilled to announce that our much-anticipated concert week is finally here! It's time to gather your friends, family, and loved ones for a week of unforgettable musical performances.<br />
+                From classical to contemporary, our lineup of talented musicians is set to entertain you with their incredible skills and passion for music. Join us as we celebrate the power of music and its ability to bring people together. <br />
+                This year's concert week promises to be a memorable one, with a range of musical genres to suit everyone's taste. Whether you prefer the soothing sounds of jazz or the upbeat rhythms of pop, we've got you covered. <br />
+                So, what are you waiting for? <br />Book your tickets now and get ready to be blown away by our amazing performers. It's a week you won't want to miss!
+                Don't forget to spread the word and invite your friends and family to join us for this incredible musical experience. <br />Let's make this concert week one to remember!
+                We can't wait to see you there!
+            </p>
+            <p id="contact">Please contact us via email incase of any further questions at <span class="fw-bold text-decoration-underline">+254concerts@kenya.com ></span>
+            </p>
+            </div>
+        </div>
+    </section>
+    <section>
+        <div class="container-fluid my-3 p-4 d-flex flex-sm-column flex-column align-items-center text-center">
+            <h2>+254 Concert Week Logo</h2>
+            <hr align="center" width="10%" size="4">
+            <p id="logo_copyright">The logo of 254 Concerts The logo features a bold, modern design that            incorporates clean lines and geometric shapes. The symbol is that of a lion, Kenya's national symbol.
+                Designed by VectorStock Designs, this logo perfectly captures our company's mission and values. We are proud to display it as a symbol of our commitment to delivering exceptional products and services to our customers.
+                Attribution: Logo design by VectorStock Designs
+            </p>
+            <img class="logo-about-page border" src="images/lion-logo.png" alt="logo">
+            <div class="line my-4"></div>
+        </div>
+    </section>
+    <section>
+        <div id="past_events" class="container-fluid my-0 d-flex flex-column align-items-center text-center">
+            <h2 id="past_title">See our past events!</h2>
+            <hr align="center" width="10%" size="4">
+            <p id="past_text">Take a look at past editions of our events held in Nakuru and Mombasa</p>
+            <div id="past_event_pics" class="d-flex flex-column flex-sm-row my-2">
+                <img id="past_pics" class="my-2" src="images/photo_1.PNG" alt="past event 2013">
+                <img id="past_pics" class="my-2" src="images/photo_2.PNG" alt="past event 2011">
+            </div>
```

Comment:
- [ ] In the mobile version, you should ensure that the bottom section images are organized in a vertical column. I have realized that this was not the case as at `700px` the cards are horizontal. Kindly make sure that the images are vertical for all the mobile version screen sizes.

Your design 😎 
![image](https://user-images.githubusercontent.com/99394374/224724164-09745019-9029-404b-becd-1f6b4b7cf499.png)

Reply:
Done